### ISSUE TITLE
Review accessibility/SEO benchmarks

### ISSUE THREAD

**Issue by WrathfulSpatula** [unitaryfund/metriq-app#657]
Upon implementing Kass' and Frances' UX design, browser accessibility and SEO benchmarks should be reviewed, for best user experience and top search engine ranking. As a result of that review, please open new issue tickets for each suggestion from those benchmarks.

**Comment by WrathfulSpatula** [comment]
We made a couple of good passes at this, already. It's something we should check regularly, but the scores are good, until we change the fundamental homepage layout.
